prac_05/wimbledon.py

```diff
@@ -0,0 +1,46 @@
+"""
+Wimbledon
+"""
+import csv
+
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    """Get data, extract champion information and display data."""
+    data = get_data()
+    champion_to_count, countries = extract_data(data)
+    display_data(champion_to_count, countries)
+
+
+def get_data():
+    """Read wimbledon file and return data."""
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        reader = csv.reader(in_file)
+        next(reader)
+        data = list(reader)
+    return data
+
+
+def extract_data(data):
+    """Extract countries in a set and champions and count to a dictionary."""
+    countries = {row[1] for row in data}
+    champion_to_count = {}
+    for row in data:
```

Comment:
row could be named better as currently it is unclear what it is supposed to represent

Reply:
I think for csv files row can be used instead of line, but I'll make sure to look into it more.